spyder/widgets/ipythonconsole/shell.py

```diff
@@ -70,6 +71,9 @@ def __init__(self, ipyclient, additional_options, interpreter_versions,
         # To save kernel replies in silent execution
         self._kernel_reply = None
 
+        # To set the color of the matched parentheses
+        self.set_bracket_matcher_color_scheme(self.syntax_style)
```

Comment:
Ok, got it.

Could you improve your comment above this line with a short summary of what you just said? Else, I'm sure me or someone else will be tempted to remove this line in the future ;-)